src/pfe/portal/routes/projectTypes.route.js

```diff
@@ -11,10 +11,13 @@
 const express = require('express');
 
 const Logger = require('../modules/utils/Logger');
+const TemplateError = require('../modules/utils/errors/TemplateError');
 
 const router = express.Router();
 const log = new Logger(__filename);
 
+const TEMPLATES_LOCKED_STATUS_CODE = 423;
```

Comment:
could you write this constant once, and export it (probably utils)? It's duplicated across 2 routes